tfx/examples/chicago_taxi_pipeline/serving/chicago_taxi_client.py

```diff
@@ -143,7 +147,18 @@ def _do_inference(model_handle, examples_file, num_examples, schema):
         elif feature.type == schema_pb2.BYTES:
           one_example[name] = [one_line[name].encode('utf8')]
       else:
-        one_example[name] = []
+        # TF serve does not like missing features, so we'll populate
+        # missing features with mean / mode values of that feature
+        if feature.type == schema_pb2.FLOAT:
+          one_example[name] = [feature_stats.num_stats.mean]
+        elif feature.type == schema_pb2.INT:
+          one_example[name] = [int(feature_stats.num_stats.mean)]
+        elif feature.type == schema_pb2.BYTES:
+          top_values = list(feature_stats.string_stats.top_values)
+          if len(top_values) > 0:
+            one_example[name] = [top_values[0].value.encode('utf8')]
+          else:
+            one_example[name] = ["".encode('utf8')]
```

Comment:
Could you use single quotes for consistency?

Reply:
Please let me know if I need to create a new PR or what. This is my first time contributing and I don't know how to proceed. The requested change is made and pushed to my fork and I can see it below. But I don't know if I need to do something before it is part of the PR.

Reply:
This should be enough. Let me begin the internal review process!